### PR TITLE
[hotfix] Make comment language friendly to contributors

### PR DESCRIPTION
**Pull request by sjwiesman**

### CODE REVIEW

**Comment by sjwiesman** [comment]
@autophagy maybe you want to look at this as well?

File: config.yaml [hunk 1]
```diff
@@ -20,38 +20,77 @@ stale_assigned:
     stale_days: 14
     warning_days: 7
     warning_label: "stale-assigned"
-    warning_comment: 'This issue is assigned but has not received an update in {stale_days} days so it has been labeled "{warning_label}". If you are still working on the issue or are waiting for feedback from a Committer, please remove the label and optionally provide an update. If you are waiting for feedback, please consider this a reminder *to Committers/Reviewer* to get back to you. If you are no longer working on the issue, please unassign yourself so someone else may work on it. In {warning_days} days the issue will be automatically unassigned.'
     done_label: "auto-unassigned"
-    done_comment: 'This issue was marked "{warning_label}" and has not received an update in {warning_days} days. It is now automatically unassigned. If you are still working on it, you can assign it to yourself again. Please also give an update about the status of the work.'
+    warning_comment: |
+        Hi!
```
Comment (knaufk) [inline config.yaml]: Do we really need to say "Hi!"?

File: config.yaml [hunk 2]
```diff
@@ -20,38 +20,77 @@ stale_assigned:
...
+
```
Comment (knaufk) [inline config.yaml]: Have you tested that line breaks are forwarded to the comment in Jira as expected?

File: config.yaml [hunk 3]
```diff
@@ -20,38 +20,77 @@ stale_assigned:
...
-    warning_comment: 'This critical issue is unassigned and itself and all of its Sub-Tasks have not been updated for {stale_days} days. So, it has been labeled "{warning_label}". If this ticket is indeed critical, please either assign yourself or give an update. Afterwards, please remove the label. In {warning_days} days the issue will be deprioritized.'
     done_label: "auto-deprioritized-critical"
-    done_comment: 'This issue has been labeled "{warning_label}" for {warning_days} days. It is deprioritized now. If this ticket is actually critical, please raise the ticket priority again and assign yourself or revive the public discussion.'
+    warning_comment: |
+        Hi!
+
+        I am the [Flink Jira Bot|https://github.com/apache/flink-jira-bot/] and I help the community manage its development. I see this issues has been marked as Critical but is unassigned and neither itself nor its Sub-Tasks have been updated for {stale_days} days. I have gone ahead and added a "{warning_label}" to the issue". If this ticket is critical, please either assign yourself or give an update. Afterwards, please remove the label or in {warning_days} days the issue will be deprioritized.
```
Comment (knaufk) [inline config.yaml]:
```suggestion
        I am the [Flink Jira Bot|https://github.com/apache/flink-jira-bot/] and I help the community manage its development. I see this issues has been marked as Critical but is unassigned and neither itself nor its Sub-Tasks have been updated for {stale_days} days. I have gone ahead and marked it "{warning_label}". If this ticket is critical, please either assign yourself or give an update. Afterwards, please remove the label or in {warning_days} days the issue will be deprioritized.
```

**Review by knaufk — CHANGES_REQUESTED** [review]
Thanks @sjwiesman for the improvements, particularly linking to the bot every time.  Generally looks good to me, I am  not a fan of the verbosity of saying "Hi" every time.

**Comment by sjwiesman** [comment]
@knaufk thanks for the quick review. I removed the `Hi!`'s and the new lines. Let me know if you have any other concerns.

File: config.yaml [hunk 4]
```diff
@@ -20,38 +20,55 @@ stale_assigned:
...
+    warning_comment: |
+        I am the [Flink Jira Bot|https://github.com/apache/flink-jira-bot/] and I help the community manage its development. I see this issue is assigned but has not received an update in {stale_days}, so it has been labeled "{warning_label}".
+        If you are still working on the issue, please remove the label and add a comment updating the community on your progress.  If this issue is waiting on feedback, please consider this a reminder to the committer/reviewer. Flink is a very active project, and so we appreciate your patience.
+        If you are no longer working on the issue, please unassign yourself so someone else may work on it. If the "warning_label" label is not removed in {warning_days} days, the issue will be automatically unassigned.
+
+    done_comment: |
+        This issue was marked "{warning_lablel}" {warning_days} ago and has not received an update. I have automatically removed the current assignee from the issue so others in the community may pick it up. If you are still working on this ticket, please ask a committer to reassign you and provide an update about your current status.
```
Comment (NicoK) [inline config.yaml]:
```suggestion
        This issue was marked "{warning_label}" {warning_days} ago and has not received an update. I have automatically removed the current assignee from the issue so others in the community may pick it up. If you are still working on this ticket, please ask a committer to reassign you and provide an update about your current status.
```

Comment (sjwiesman) [reply config.yaml]: nice catch

File: config.yaml (hunk 3 above)
Comment (knaufk) [reply config.yaml]: @sjwiesman did you not apply this one on purpose?

Comment (sjwiesman) [reply config.yaml]: Just an oversight. I've applied the change.

**Review by knaufk — APPROVED** [review]
LGTM

**Pull request closed without merging**